src/repositories/statusReasonRepository.test.ts

```diff
@@ -0,0 +1,43 @@
+import { assert } from 'chai';
+import { ProjectStatusReason } from '../entities/projectStatusReason';
+import {
+  findAllStatusReasons,
+  findStatusReasonsByStatusId,
+} from './statusReasonRepository';
+import { ProjectStatus } from '../entities/projectStatus';
+import { ProjStatus } from '../entities/project';
+
+describe('findAllStatusReasons test cases', () => {
+  it('should find all status reasons', async () => {
+    const status = await ProjectStatus.findOne({
+      id: ProjStatus.active,
+    });
+    await ProjectStatusReason.create({
+      status,
+      description: 'test',
+    }).save();
+    const allStatusReasons = await findAllStatusReasons();
+    assert.isOk(allStatusReasons);
+    assert.notEqual(allStatusReasons.length, 0);
```

Comment:
Instead of notEqual, to make the test direct and clear you should do isAbove.
```javascript
assert.isAbove(allStatusReasons.length, 0)
```

Reply:
There is huge list of assertions we can use in test with many functionalities: https://www.chaijs.com/api/assert/

Reply:
You are right, I found better assertion in the link that you sent, and changed to 
    `assert.isNotEmpty(allStatusReasons);`
instead of put assertion on length of arra